config/locales/en.yml

```diff
@@ -1,8 +1,41 @@
 ---
 en:
   decidim:
+    admin:
+      actions:
+        confirm_destroy: Are you sure you want to delete this %{name}?
+        destroy: Delete
+        edit: Edit
+        preview: Preview
```

Comment:
This items should scoped under `decidim.anonymous_codes.admin`

Reply:
Done